internal/cmd/policy/listprincipals/listprincipals.go

```diff
@@ -0,0 +1,79 @@
+// Copyright The gittuf Authors
+// SPDX-License-Identifier: Apache-2.0
+
+package listprincipals
+
+import (
+	"fmt"
+	"strings"
+
+	"github.com/gittuf/gittuf/experimental/gittuf"
+	"github.com/gittuf/gittuf/internal/tuf"
+	"github.com/spf13/cobra"
+)
+
+const indentString = "    "
+
+type options struct {
+	targetRef  string
+	policyName string
+}
+
+func (o *options) AddFlags(cmd *cobra.Command) {
+	cmd.Flags().StringVar(
+		&o.targetRef,
+		"target-ref",
+		"policy",
+		"specify which policy ref should be inspected",
```

Comment:
renamed this to policy-ref to be clearer. Good call, it's still target-ref in list-rules. I'd argue we want to just make it a boolean `--staging` flag since we're not actively working on having multiple "feature refs" for policies. cc @neilnaveen who had some ideas here.

Reply:
https://github.com/gittuf/gittuf/compare/f186ef406a8118b8eb3d9eadbca745ab0a14018d..20d37fbd0ce52b91963f4b3d8aa047af760c5c6f